example/LLM/bard/text_cls/code/evaluator.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+from Bard import Chatbot
+
+from starwhale import evaluation, multi_classification
+
+# https://github.com/acheong08/Bard
+# This evaluation shall be run in USA or UK, or errors would be raised
+bard_key = os.getenv("BARD_TOKEN")
+chatbot = Chatbot(bard_key)
+
+
+_LABEL_NAMES = ["world", "sports", "business", "sci/tech"]
+
+
+def label_number(raw: str) -> None:
```

Comment:
return int, but the typing return `None` ?

Reply:
done